src/types.ts

```diff
@@ -154,6 +154,10 @@ export type AssetTransferApiOpts<T extends ChainInfoKeys | InjectedChainInfoKeys
 	 * Option to inject chain information into the registry.
 	 */
 	injectedRegistry?: RequireAtLeastOne<ChainInfoRegistry<T>>;
+	/**
+	 * Option to override the registry with the supplied chain information.
+	 */
+	overridingRegistry?: RequireAtLeastOne<ChainInfoRegistry<InjectedChainInfoKeys>>;
```

Comment:
To keep things in the active tense, I think we should call this `overrideRegistry`.